src/server/routes/embed.js

```diff
@@ -91,7 +91,7 @@ module.exports = async (req: $Request, res: $Response) => {
 
         <style>
           html {
-            font-size: 16px;
+            font-size: 14px;
```

Comment:
Was this change meant to be included? Does not seem relevant

Reply:
This is not to be there. fixing that.

Reply:
done https://github.com/Paratii-Video/paratii-portal/pull/301/commits/6baf7d4bf6bd521621ce67a64a8ea4bbee0f34e8